src/components/PRGroupDetail.tsx

```
@@ -17,106 +18,169 @@ export function PRGroupDetail({ group, onBack }: PRGroupDetailProps) {
-        <button onClick={onBack} className="btn btn-ghost gap-2">
-          <svg xmlns="http://www.w3.org/2000/svg" className="h-5 w-5" viewBox="0 0 20 20" fill="currentColor">
-            <path fillRule="evenodd" d="M9.707 16.707a1 1 0 01-1.414 0l-6-6a1 1 0 010-1.414l6-6a1 1 0 011.414 1.414L5.414 9H17a1 1 0 110 2H5.414l4.293 4.293a1 1 0 010 1.414z" clipRule="evenodd" />
-          </svg>
-          Voltar
-        </button>
-      </div>
+    <div className="w-full">
+      <div className="container mx-auto px-4 py-6 max-w-7xl">
+        {/* Back Button */}
+        <div className="mb-6">
+          <button onClick={onBack} className="btn btn-ghost gap-2">
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)